install/yaml/install.yaml

```diff
@@ -5672,7 +5672,7 @@ spec:
                   type: string
                   minLength: 1
                   maxLength: 63
-                  pattern: "^[a-z0-9]([-\\.a-z0-9]*[a-z0-9])?$"
+                  pattern: "^[a-z0-9]([-\\.a-z0-9]*[a-z0-9])?$"                
```

Comment:
Is this change needed or is this just some extra spacing ?

Reply:
It's an artifact of the `{{- include` - not clear how easy it is to remove but I've seen it in other cases. (I'm happy to ignore it.)

Reply:
okok sounds good!